pkgs/data/icons/catppuccin-cursors/default.nix

```
@@ -15,60 +13,39 @@ let
   variantName = { palette, color }: (lib.strings.toLower palette) + color;
   variants = map variantName product;
 in
-stdenvNoCC.mkDerivation {
+stdenvNoCC.mkDerivation rec {
   pname = "catppuccin-cursors";
-  version = "unstable-2022-08-23";
+  version = "0.2.0";
+  dontBuild = true;
 
   src = fetchFromGitHub {
     owner = "catppuccin";
     repo = "cursors";
-    rev = "3d3023606939471c45cff7b643bffc5d5d4ff29c";
-    sha256 = "1z9cjxxsj3vrmhsw1k05b31zmncz1ksaswc4r1k3vd2mmpigq1nk";
+    rev = "v${version}";
+    sha256 = "sha256-pz14QY8OMPioQSlXhENPWrxg2rjJ6PrzsCi51e1AxIE=";
   };
 
+  buildInputs = [unzip];
+
   outputs = variants ++ [ "out" ]; # dummy "out" output to prevent breakage
 
   outputsToInstall = [];
 
-  nativeBuildInputs = [
-    inkscape
-    xcursorgen
-  ];
-
-  postPatch = ''
-    patchShebangs ./build.sh
-  '';
-
-  # Make fontconfig stop warning about being unable to load config
-  FONTCONFIG_FILE = makeFontsConf { fontDirectories = [ ]; };
-
-  # Make inkscape stop warning about being unable to create profile directory
-  preBuild = ''
-    export HOME="$NIX_BUILD_ROOT"
-  '';
-
   installPhase = ''
-    runHook preInstall
-
-    for output in $outputs; do
+      for output in $outputs; do
       if [ "$output" != "out" ]; then
         local outputDir="''${!output}"
         local iconsDir="$outputDir"/share/icons
-
         mkdir -p "$iconsDir"
-
         # Convert to kebab case with the first letter of each word capitalized
         local variant=$(sed 's/\([A-Z]\)/-\1/g' <<< "$output")
         local variant=''${variant^}
-
-        cp -r dist/Catppuccin-"$variant"-Cursors "$iconsDir"
+        unzip cursors/Catppuccin-"$variant"-Cursors.zip -d "$iconsDir"
       fi
     done
 
-    # Needed to prevent breakage
-    mkdir -p "$out"
-
-    runHook postInstall
+    # prevent the script breaking
+    mkdir $out
```

```suggestion
    # Needed to prevent breakage
    mkdir "$out"

    runHook postInstall
```
Same as above, and keeps the diff minimal as well.